flask-app/openapi_server/templates/home.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
This page may be removed later and we will deny requests sent to the root path.

Reply:
Not sure why ? It provides a quick test for Dev-ops and us to see the container is running and does not  introduce a security risk.

Reply:
It's fine to keep the page but we can make it look nicer in the future, e.g. showing information from `/health`.